libcudacxx/include/cuda/__data_movement/property_traits.h

```diff
@@ -0,0 +1,123 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of CUDA Experimental in CUDA C++ Core Libraries,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef __CUDAX_CUDA_LOAD_STORE_PROPERTY_TRAITS
+#define __CUDAX_CUDA_LOAD_STORE_PROPERTY_TRAITS
+
+#include <cuda/__cccl_config>
+
+#if defined(_CCCL_IMPLICIT_SYSTEM_HEADER_GCC)
+#  pragma GCC system_header
+#elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_CLANG)
+#  pragma clang system_header
+#elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_MSVC)
+#  pragma system_header
+#endif // no system header
+
+#include <cuda/std/__type_traits/integral_constant.h>
+
+#include <cuda/experimental/__mdspan/properties.cuh>
+
+namespace cuda::experimental
+{
+
+/***********************************************************************************************************************
+ * Load Behavior
+ **********************************************************************************************************************/
+
+template <typename>
+struct is_memory_behavior : ::cuda::std::false_type
+{};
+
+template <MemoryBehavior Value>
+struct is_memory_behavior<memory_behavior_t<Value>> : ::cuda::std::true_type
+{};
+
+template <typename T>
+constexpr bool is_memory_behavior_v = is_memory_behavior<T>::value;
```

Comment:
With always having variable templates at hand we always want to specialize the variable template and if we add it derive the struct from it
```suggestion
template <typename T>
inline constexpr bool is_memory_behavior_v = is_memory_behavior<T>::value;
```